cmd/root.go

```diff
@@ -383,19 +391,46 @@ func updateIssue(ghIssue github.Issue, jIssue jira.Issue, ghClient github.Client
 		fields.Unknowns[key] = time.Now().Format(dateFormat)
 
 		fields.Type = jIssue.Fields.Type
-		fields.Summary = jIssue.Fields.Summary
+		if fields.Summary == "" {
+			fields.Summary = jIssue.Fields.Summary
+		}
 
 		issue := &jira.Issue{
 			Fields: &fields,
 			Key:    jIssue.Key,
 			ID:     jIssue.ID,
 		}
 
-		issue, res, err := jClient.Issue.Update(issue)
+		if !dryRun {
+			_, res, err := jClient.Issue.Update(issue)
 
-		if err != nil {
-			log.Errorf("Error updating JIRA issue %s: %v", jIssue.Key, err)
-			return getErrorBody(res)
+			if err != nil {
+				log.Errorf("Error updating JIRA issue %s: %v", jIssue.Key, err)
+				return getErrorBody(res)
+			}
+		} else {
+			log.Info("")
+			log.Infof("Update JIRA issue %s with GitHub issue #%d:", jIssue.Key, ghIssue.GetNumber())
+			if fields.Summary != jIssue.Fields.Summary {
+				log.Infof("  Summary: %s", fields.Summary)
+			}
+			if fields.Description != "" {
+				fields.Description = newlineReplaceRegex.ReplaceAllString(fields.Description, "\\n")
+				if len(fields.Description) > 20 {
+					log.Infof("  Description: %s...", fields.Description[0:20])
```

Comment:
The fields all have whitespace for clarity. All the dry-runs print:

Doing action
  Field1: ...
  Field2: ...